src/components/DocumentUtility/DocumentUtility.tsx

```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React, { useState } from "react";
 import styled from "@emotion/styled";
```

Comment:
want to try writing a simple integration test for this? (check if logo exists within qr code)

Reply:
Sure!